lib/wasix/src/bin_factory/binary_package.rs

```diff
@@ -60,35 +63,18 @@ impl BinaryPackageCommand {
 
     /// Get the WebAssembly features required by this command's module
     pub fn wasm_features(&self) -> Option<wasmer_types::Features> {
-        // Create default features
-        let mut features = wasmer_types::Features::default();
-
-        // Try to extract information from any annotations that might exist
-        if let Some(wasm_anno) = self.metadata().annotations.get("wasm") {
-            tracing::debug!("Found wasm annotation: {:?}", wasm_anno);
-            // We found an annotation, enable the basic features by default
-            features.reference_types(true);
-            features.bulk_memory(true);
-
-            // We'll use a simpler approach - if we have wasm annotations, just enable
-            // the features that are commonly needed for most modules
-            features.exceptions(true);
-
-            tracing::debug!("WebC module features from annotations: {:?}", features);
-            return Some(features);
+        // Return only the pre-computed features from the container manifest
+        if let Some(features) = &self.features {
+            tracing::debug!(
+                "Using pre-computed WebAssembly features from annotations: {:?}",
+                features
+            );
```

Comment:
Not needed, too verbose and only needed for the PR. If any logging, we should do it at the moment of reading the webc with a serialization of the whole manifest